waku/waku_sync/storage.nim

```diff
@@ -0,0 +1,59 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import std/[times, tables], chronicles, chronos, stew/results
+
+import ./raw_bindings
+
+logScope:
+  topics = "waku sync"
+
+type WakuSyncStorageManager* = object
+  storages: OrderedTable[string, Storage]  # Map of dateTime and Storage objects. DateTime is of the format YYYYMMDDHH
```

Comment:
string are unwieldy lets use a timestamp instead.

Reply:
Agreed, Will figure out how to use timestamp with only till hour format.